### PR TITLE
LPD-27049 Add specific names for input ids on workflow node information section

### PR DESCRIPTION
**Pull request by danvb112**
Related Issue: https://liferay.atlassian.net/browse/LPD-27049

### CODE REVIEW

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/test/js/designer/definition-builder/diagram-builder/components/sidebar/sections/NodeInformationBaseSection.spec.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {render} from '@testing-library/react';
+
+import '@testing-library/jest-dom/extend-expect';
+import React from 'react';
+
+import {NodeInformationBaseSection} from '../../../../../../../../src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/NodeInformationBaseSection'; // Adjust path as needed
```
Comment (rebsilva) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/test/js/designer/definition-builder/diagram-builder/components/sidebar/sections/NodeInformationBaseSection.spec.js:11]: hi @danvb112, this comment should be here?

**Review by rebsilva — CHANGES_REQUESTED** [review]
great work @danvb112! just one question and it's good to go :checkered_flag:

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/test/js/designer/definition-builder/diagram-builder/components/sidebar/sections/NodeInformationBaseSection.spec.js (hunk 1 above)
Comment (danvb112) [reply modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/test/js/designer/definition-builder/diagram-builder/components/sidebar/sections/NodeInformationBaseSection.spec.js]: Ops, my bad! Thanks @rebsilva!

**Comment by danvb112** [comment]
ci:test:sf

**Comment by danvb112** [comment]
ci:test:relevant

**Comment by danvb112** [comment]
ci:test:workflow

**Comment by danvb112** [comment]
Sent to: https://github.com/liferay-workflow/liferay-portal/pull/1238

**Pull request closed without merging**